dspace-server-webapp/src/main/java/org/dspace/app/rest/security/EntityTypeRestPermissionEvaluator.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.security;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.model.EntityTypeRest;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EntityTypeRestPermissionEvaluator extends RestObjectPermissionEvaluatorPlugin {
```

Comment:
This needs JavaDocs too, and also should likely have an `EvaluatorPlugin` suffix (just like other such Plugins)